core/src/main/scala/org/bitcoins/core/api/wallet/WalletApi.scala

```diff
@@ -396,6 +396,8 @@ trait WalletApi extends StartStopAsync[WalletApi] {
     * is a change output from this wallet
     */
   def isChange(output: TransactionOutput): Future[Boolean]
+
+  def getSyncState(): Future[(Int, DoubleSha256DigestBE)]
```

Comment:
So this LGTM, but one thing that might be nice is to rename our current `FilterSyncMarker` to just `SyncMarker` and then use that here too? 

https://github.com/bitcoin-s/bitcoin-s/blob/2de17bb4e47961fd3fe4e55d75909b42470e3c59/core/src/main/scala/org/bitcoins/core/api/chain/FilterSyncMarker.scala#L11

Reply:
It looks like we've had this discussion before, are the contexts the same? https://github.com/bitcoin-s/bitcoin-s/pull/2003#issuecomment-691176472

Reply:
I don't think it makes sense to use `FilterSyncMarker` or renaming it. `FilterSyncMarker`'s height and hash aren't meant to refer to the same block

Reply:
ok, then possibly make a `BlockSyncMarker` with appropriate field names?